changelogs/fragments/62518-nobacktracefix_vyos.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - https://github.com/ansible/ansible/pull/62518      
```

Comment:
Could you please add some information about the fix here, this will help end-user to identify changes back ported ?

Reply:
Added , Thanks !